src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -460,6 +461,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
     // Allow FP16 Converts to be folded and FP16 constants to be upgraded to FP32 data type
     pass_config->disable<ov::pass::DisableDecompressionConvertConstantFolding>();
     pass_config->disable<ov::pass::ConvertCompressedOnlyToLegacy>();
+    pass_config->disable<ov::pass::EyeDecomposition>();
```

Comment:
The CPU plugin doesn't support decomposition for all cases? The is possibility for adding cases when pass shouldn't be applied

Reply:
It's not like CPU doesn't support decomposition, CPU support all of the Eye cases, so decomposition is not needed for CPU at all;)